plugins/UFM_NDT_Plugin/build/config/ndt.conf

```diff
@@ -15,4 +15,8 @@ host_patterns: (^(SAT|DSM)\d+ ibp.*$)
 # interval in seconds to verify boundary port state after topoconfig deployment
 port_validation_sleep_interval=5
 # number of attempts to verify boundary port state - polling
-port_validation_number_of_attempts=5
\ No newline at end of file
+port_validation_number_of_attempts=5
+# Address of server to be used to send cable validation report request. If not set - feature not enabled.
```

Comment:
missing explanation about local host and remote host options - and that for local host no credentials are needed and for remote server it is required.

Reply:
no credentials in conf file. what is relation for credentials in this conf file - only host name and port number that is set using rest api